packages/strapi-plugin-upload/services/image-manipulation.js

```diff
@@ -126,6 +152,7 @@ const generateBreakpoint = async (key, { file, breakpoint }) => {
     width: breakpoint,
     height: breakpoint,
     fit: 'inside',
+    format: file.mime.split('/')[1],
```

Comment:
This could be more strict. Using [mime-types](https://www.npmjs.com/package/mime-types) to validate the extension – **mime.extension(type)**

Not sure what the recommendation is on adding new packages.

Reply:
Also, not entirely sure the downstream effects that adding `format` to this object has.